ehrql/docs/language.py

```diff
@@ -75,6 +75,7 @@ def build_language():
         "dataset": dict(
             create_dataset=namespace["create_dataset"],
             Dataset=namespace["Dataset"],
+            get_dataset_parameter=namespace["get_dataset_parameter"],
```

Comment:
I wonder if this belongs in it's own section, "Parameters" or something. It's not specific to just datasets.